book.rb

```diff
@@ -0,0 +1,11 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  has_many :rentals
+  has_many :persons, through: :rentals
```

Comment:
- [ ] It will be great to delete these methods. This is because they are actually used in `active-record associations` in the rails framework. Since this is just pure ruby, they are not needed